packages/vats/src/bootstrap.js

```diff
@@ -577,6 +648,8 @@ export function buildRootObject(vatPowers, vatParameters) {
                   }
                   // We have an unusedPayment, so we can split off a payment.
                   const amount = AmountMath.make(brand, Nat(value));
+                  // TODO: what happens if unusedBankPayment doesn't have enough
+                  // remaining?
```

Comment:
It looks like we fail safe in this case (i.e. the whole thing comes crashing down) so I'm not checking for an issue.